packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/BriefcaseStatus.tsx

```diff
@@ -33,12 +33,9 @@ export const BriefcaseStatus = ({
   onDownloadClick,
   className,
 }: BriefcaseStatusProps) => {
-  const [title, setTitle] = useState<string>();
-  const [classNames, setClassNames] = useState<string>();
-
   const isDownloading = !!mergeProgress && mergeProgress < 100;
 
-  useEffect(() => {
+  const getTitleAndClass = useCallback(() => {
```

Comment:
im wondering if we should move the class names into the components below

wonder if theres a prop on the icons we can use to denote its not actionable / aka disable